astro/src/pages/team/partners.astro

```diff
@@ -25,14 +25,14 @@ const crumbs = [{
 <Layout title={title} crumbs={crumbs} heading={heading}>
  <Section theme="body-tertiary">
   <p class="lead mb-5">Thank you to the organizations that partner with us and support our work in various ways.</p>
-  <ul class="row list-unstyled text-center">
-    <li class="col-md-4 p-4"><a href="https://www.deque.com/" target="_blank"><Image src={dequeLogo} width="150" alt="Deque"></a></li>
-    <li class="col-md-4 p-4 partner-logo"><a href="https://knowbility.org/" target="_blank"><Image src={knowbilityLogo} width="150" alt="Knowbility"></a></li>
-    <li class="col-md-4 p-4 partner-logo"><a href="https://www.levelaccess.com/" target="_blank"><Image src={levelAccessLogo} width="150" alt="Level Access"></a></li>
-    <li class="col-md-4 p-4 partner-logo"><a href="https://www.pearson.com/en-us.html" target="_blank"><Image src={pearsonLogo} width="150" alt="Pearson"></a></li>
-    <li class="col-md-4 p-4 partner-logo"><a href="https://teachaccess.org/" target="_blank"><Image src={teachAccessLogo} width="150" alt="Teach Access"></a></li>
-    <li class="col-md-4 p-4 partner-logo"><a href="https://tetralogical.com/" target="_blank"><Image src={tetraLogicalLogo} width="150" alt="TetraLogical"></a></li>
-    <li class="col-md-4 p-4 partner-logo"><a href="https://www.tpgi.com/" target="_blank"><Image src={tpgiLogo} width="150" alt="TPGi"></a></li>
+  <ul class="row list-unstyled text-center shadow-lg bg-light-secondary rounded">
```

Comment:
Let's try to find a medium gray that can work for dark mode as the current background is too light. It'll be a challenge with some of the coloring of the logos.